src/Backend/Modules/Pages/Actions/Settings.php

```diff
@@ -22,32 +33,95 @@ public function execute(): void
+            $data = $this->settingsForm->getData();
+
+            // set our settings
+            $this->get('fork.settings')->set(
+                $this->getModule(),
+                'meta_navigation',
+                $data->metaNavigation
+            );
+
+            $this->get('fork.settings')->set(
+                $this->getModule(),
+                'offline_title_' . Language::getWorkingLanguage(),
+                $data->offlineTitle
+            );
+
+            $this->get('fork.settings')->set(
+                $this->getModule(),
+                'offline_text_' . Language::getWorkingLanguage(),
+                $data->offlineText
+            );
+
+            if ($data->offlineImage instanceof MediaGroup) {
+                $this->get('media_library.repository.group')->add($data->offlineImage);
+                $this->get('doctrine.orm.entity_manager')->flush();
                 $this->get('fork.settings')->set(
                     $this->getModule(),
-                    'meta_navigation',
-                    (bool) $this->form->getField('meta_navigation')->getValue()
+                    'offline_image_' . Language::getWorkingLanguage(),
+                    $data->offlineImage->getId()
                 );
-
-                // redirect to the settings page
-                $this->redirect(BackendModel::createUrlForAction('Settings') . '&report=saved');
             }
+
+            $this->renderOfflinePage();
+
+            // redirect to the settings page
+            $this->redirect(BackendModel::createUrlForAction('Settings') . '&report=saved');
         }
     }
+
+    private function renderOfflinePage(): void
+    {
+        /** @var Environment $twig */
+        $twig = $this->get('twig');
+
+        /** @var ParseBlocksExtension $extension */
+        $extension = $twig->getExtension(ParseBlocksExtension::class);
+
+        $html = $twig->render(
+            BACKEND_MODULES_PATH . '/' . $this->getModule() .'/Layout/Templates/Offline.html.twig',
+            [
+                'title' => $this->get('fork.settings')->get(
+                    $this->getModule(),
+                    'offline_title_' . Language::getWorkingLanguage()
+                ),
+                'text' => $extension->parseBlocks(
+                    $this->get('fork.settings')->get(
+                        $this->getModule(),
+                        'offline_text_' . Language::getWorkingLanguage()
+                    )
+                )
+                 ,
+                'image' => $this->get('media_library.repository.group')->findOneById(
+                    $this->get('fork.settings')->get(
+                        $this->getModule(),
+                        'offline_image_' . Language::getWorkingLanguage()
+                    )
+                )
+            ]
+        );
+
+        $offlinePage = fopen(OFFLINE_PATH . '/offline_' . Language::getWorkingLanguage() . '.html', 'w');
```

Comment:
Please use Filesystem  dump of symfony for this.
Looks like the page is generated but you committed the generated code. Can you have a look at removing it?
Is this also enabled by default? If so if it generated when installing fork?